prometheus/utils.go

```diff
@@ -26,10 +27,17 @@ func connectRaw(ctx context.Context, cc *connection.ConnectionCache, c *plugin.C
 
 	// Prefer config settings
 	prometheusConfig := GetConfig(c)
+
+	baseUrl := os.Getenv("PROMETHEUS_URL")
```

Comment:
@beudbeud In our plugins, we usually have the config arg take precedence over environment variables, e.g., https://github.com/turbot/steampipe-plugin-crowdstrike/blob/main/crowdstrike/client.go#L32-L36.

Can you please update the code so the `address` config arg will be used if both are set?